### PR TITLE
refactored getFeatureInfo

### PR DESCRIPTION
**Pull request by JowieXiang**
sorted out *getFeatureInfo* feature from *Transerve's* PR #159

### CODE REVIEW

File: webapp/views/map/betterWMS/L.TileLayer.BetterWMS.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    html += "<td>"+value+"</td></tr>"
+  }
+
+  html = html + "</tbody></table></div>";
+  return html;
+}
+
+L.tileLayer.betterWms = function (url, options) {
+  return new L.TileLayer.BetterWMS(url, options);  
+};
```
Comment (janbehrens) [inline webapp/views/map/betterWMS/L.TileLayer.BetterWMS.js:92]: There are 38 eslint problems in this file.

**Review by janbehrens — CHANGES_REQUESTED** [review]
Suggestions:
- open popup on click instead of doubleclick
- make the popup larger and improve its style
- don't show "cat" and tags with no value

**Comment by JowieXiang** [comment]
> Suggestions:
> 
> * open popup on click instead of doubleclick
> * make the popup larger and improve its style
> * don't show "cat" and attributes with no value

Added corresponding changes in the new commits.

File: webapp/views/map/script.js [hunk 2]
```diff
@@ -102,5 +102,5 @@ function refreshLayer(layer) {
  * @param {object} service config object from config.js
  */
 function createWms(service) {
-  return service.type === 'vector' ? L.tileLayer.wms(vectorWMS, service) : L.tileLayer.wms(rasterWMS, service)
+  return service.type === 'vector' ? L.tileLayer.betterWms(vectorWMS, service) : L.tileLayer.betterWms(rasterWMS, service)
```
Comment (janbehrens) [inline webapp/views/map/script.js:105]: Isn't it quite useless for raster layers?

**Review by janbehrens — APPROVED** [review]

**Pull request merged**